logfile_to_pcap_convertor/src/logfile_to_pcap_convertor.dep

```diff
@@ -0,0 +1,67 @@
+logfile_to_pcap_convertor.o: src/logfile_to_pcap_convertor.c \
```

Comment:
This file should be deleted / removed from the repo. It's generated during building.